src/frontend/js/components/DashboardModerators/ListModerators/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { User } from '../../../types/User';
+import { Moderator } from '../Moderator';
+import { messagesDashboardModerators } from '../messages';
+import { ListUserProps } from '../SelectStudentsSuggestField';
+
+export interface ListModeratorProps extends ListUserProps {
```

Comment:
```suggestion
export interface ListModeratorsProps extends ListUserProps {
```

And I don't think it's a good practice to extends component props from an other component props. Specially in that case. this component is not a child of `SelectStudentsSuggestField` component.

Reply:
When it was in the same file, I thought it was a good idea in order not to repeat yourself, now I don't see the point anymore for comprehension.